Assets/AppCenter/Plugins/AppCenterSDK/Distribute/Android/DistributeDelegate.cs

```diff
@@ -30,7 +30,11 @@ bool onReleaseAvailable(AndroidJavaObject activity, AndroidJavaObject details)
 
         void onNoReleaseAvailable(AndroidJavaObject activity)
         {
-            Distribute.NoReleaseAvailable?.Invoke();
+            if (Distribute.NoReleaseAvailable == null)
```

Comment:
Do we need this change?

Reply:
Due to build error `Feature 'null propagating operator' cannot be used because it is not part of the C# 4.0 language specification.`